modules/install-sno-monitoring-the-installation-manually.adoc

```diff
@@ -4,22 +4,28 @@
 
 :_content-type: PROCEDURE
 [id="install-sno-monitoring-the-installation-manually_{context}"]
-= Monitoring the installation manually
+= Monitoring the cluster installation using openshift-install
 
-If you created the ISO manually, use this procedure to monitor the installation.
+Use `openshift-install` to monitor the progress of the single-node cluster install.
```

Comment:
> Do not use a word as a part of speech that is different from its standard dictionary definition. In particular, do not use verbs as nouns or adjectives; examples are configure, compile, debug, fix, and install.

You have several questionable instances of "install" in this PR.

Reply:
fixed